test/fork.js

```diff
@@ -48,8 +48,8 @@ test('exit after tests are finished', function (t) {
 
 	fork(fixture('long-running.js'))
 		.on('exit', function () {
-			t.ok(Date.now() - start < 10000, 'did NOT wait for setTimeout(fn, 15000');
-			t.ok(cleanupCompleted, 'did wait for onExit(fn) to complete');
+			t.ok(Date.now() - start < 10000, 'should NOT wait for setTimeout(fn, 15000)');
+			t.ok(cleanupCompleted, 'should wait for onExit(fn) to complete');
```

Comment:
These descriptions still look like test descriptions. Those messages are error messages, not assert descriptions. imho smth like this would be better: `t.ok(cleanupCompleted, 'clean up did not complete')`.

Would be great, if you improved those ;)